fala/templates/adviser/results.html

```diff
@@ -57,14 +57,17 @@ <h2 class="govuk-body-l govuk-!-font-weight-bold">{{ item.organisation.name }}</
             {% endif %}
             {% if item.categories|length %}
             <div class="categories">
-              <h3 class="govuk-heading-s govuk-!-margin-bottom-2">{{ _('Categories') }}</h3>
+              <h3 class="govuk-heading-s govuk-!-margin-bottom-2">{{ _('Categories ') }}</h3>
               <ul class="govuk-list govuk-list--bullet">
                 {% for cat in item.categories %}
-                  <li>{{ cat }}</li>
+                  <li >{{ cat }}</li>
                 {% endfor %}
               </ul>
             </div>
             {% endif %}
+            <a class="govuk-link" class="url" target="_blank" rel="noopener" href="https://www.google.com/maps/search/?{{ item.location|google_map_params|urlencode }}">
```

Comment:
**Optional** I'd want your take on this. But I feel `https://www.google.com/maps/search/?` shouldn't be in a template, but in a variable accessible in the template.

Reply:
This felt the simplest way of doing it. The template already knows this is a search (the filter function returns parameters for a search) so adding complexity didn't feel worth it

Reply:
I thought about google_maps_link but then wanted the URL encoding out of that method, so this seemed to be a good compromise